lib/monitor.js

```diff
@@ -121,31 +121,37 @@ internals.Monitor.prototype.start = function (callback) {
         };
     };
 
+    var reportersIndex = 0;
+
     Items.serial(this.settings.reporters, function (item, next) {
 
-        var reporter;
+        var hint = self.settings.reporters.length > 1 ? '(' + reportersIndex++ + ')' : '';
```

Comment:
I'd rather check the name earlier so you can use that in messaging throughout this block. That is pretty much the whole point of adding name attributes.